packages/exchange-analytics/test/integration/funding-rates/fixtures.ts

```diff
@@ -0,0 +1,85 @@
+import nock from 'nock'
+
+import { Provider as ProviderName, providerToBaseUrl } from '../../../src/api/constants'
+
+const baseUrlBinance = providerToBaseUrl.get(ProviderName.BINANCE) as string
+const baseUrlBybit = providerToBaseUrl.get(ProviderName.BYBIT) as string
+const baseUrlFtx = providerToBaseUrl.get(ProviderName.FTX) as string
+
+const BinanceFundingRatesGetPayload = {
+  symbol: 'ETHUSDT',
+  markPrice: '1461.20000000',
+  indexPrice: '1461.88822394',
+  estimatedSettlePrice: '1461.84099962',
+  lastFundingRate: '0.00007817',
+  interestRate: '0.00010000',
+  nextFundingTime: 1663344000000,
```

Comment:
Optionally you can replace it by `1663344000001` and check that the snapshot does not contain the decimal place. Forget it if you'll test it in models